### DISCUSSION TITLE
Competition?

### DISCUSSION THREAD

**Discussion by ArturoTrailblazer** [Sundae-Shop-Consulting/Volunteers#44] [General]
Ryan Rucker is building a volunteer management solution for NPC as an alternative to V4S - https://drive.google.com/file/d/19IZUddZrKIV9HriN4van-phK4NZu_zDy/view?pli=1

He's reached out to me to ask about the solution so just something to have to the radar.

**Comment by lmeerkatz** [comment]
Cool! It would be great to see a demo if they're up for it. I'm happy to see alternatives available.
